### PR TITLE
To-Do List - collaborative project

### PR DESCRIPTION
**Pull request by uryelah**
**Hi Tse!** 👋 

Can you please check our Todo List project and provide feedback?

**Summary:**
- applied functional programming (factories and modules) 
-  todo item contains title, description, due date,  priority, time, tags, checklist and button to add them;
- projects and todo lists are separated;
- application logic is separated to DOM manipulation;
- on the user interface is possible to see all projects, all todos in each project;
- user can assign todo to the project by selecting a project in which to create it
- todos and projects can be edited and deleted;
- todos can be completed, incompleted and expired;
- tasks can be marked as completed;
- todo can be set by day and by time;
- users can see the next task;
- users can see tasks ordered by date or by priority.

Thank you in advance for your time.

Kind regards,
**Sarah & Thaís 🦄**

### CODE REVIEW

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUIRED**

Hi Team everything looks really good in the code but for some reason, I could not test this the buttons do not click 
And please delete console.log from the code base :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

**Comment by uryelah** [comment]
We couldn't find any `console.log` in the todolist branch, in which file and line did you find one?

**Comment by uryelah** [comment]
After redownloading the repo, the main.js was not updated and contained `console.log`s. Re-bundled the app and added installation instructions on running `npm run bundle` to the readme file.

File: README.md [hunk 1]
```diff
@@ -4,61 +4,74 @@
...
-* CSS3
-* Vanila Javascript
-* Webpack
+
+- HTML
+- CSS3
+- Vanila Javascript
+- Webpack
 
 <!-- CONTACT -->
```
Comment (mcrd25) [inline README.md:72]: [OPTIONAL]
Don't forget to add Live Demo Link to README

**Review by mcrd25 — APPROVED** [review]
# Changes Approved 💪 🎊 

Hey @uryelah! Hey @thsvr! 👋 

Great job with the project! You've met all the requirements.
I like the effort.

Maybe you can work on some of your more wonky features, but really this is a really creative and clearly thoughtful project you did here!

***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

**Pull request merged**